Assets/UI/WorldView/citybannermanager_CQUI.lua

```diff
@@ -115,8 +115,14 @@ function CQUI_Common_CityBanner_Initialize(self, playerID, cityID, districtID, b
         and bannerStyle == BANNERSTYLE_LOCAL_TEAM 
         and playerID == Game.GetLocalPlayer()) then
         -- Register the callbacks 
-        self.m_Instance.CityBannerButton:RegisterCallback( Mouse.eMouseEnter, CQUI_OnBannerMouseOver );
-        self.m_Instance.CityBannerButton:RegisterCallback( Mouse.eMouseExit,  CQUI_OnBannerMouseExit );
+        self.m_Instance.CityNameButton:RegisterCallback( Mouse.eMouseEnter, CQUI_OnBannerMouseOver );
+        self.m_Instance.CityNameButton:RegisterCallback( Mouse.eMouseExit,  CQUI_OnBannerMouseExit );
+		self.m_Instance.CityNameButton:SetVoid1(playerID);
+		self.m_Instance.CityNameButton:SetVoid2(cityID);
```

Comment:
```suggestion
```
Unnecessary function calls since we have them below as well. They only need to be set once. Tried it without these 2 lines and it's working the same way.